tests/test_rooms/test_room_analytics.py

```diff
@@ -136,3 +139,17 @@ def test_analytics_for_monthly_bookings_diff_years(self, mock_get_json):
         CommonTestCases.admin_token_assert_equal(
             self, get_bookings_count_monthly_diff_years,
             get_bookings_count_monthly_diff_years_response)
+
+    def test_single_room_daily_bookings(self, mock_get_json):
```

Comment:
Good stuff @dnuwa , Kindly include the docstrings for these tests methods.

Reply:
@ThaDeveloper this has been worked on.